lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -449,9 +449,11 @@ def create_xcconfig_file(native_target, resource_bundle_targets)
           # @return [void]
           #
           def create_test_xcconfig_files(test_native_targets, test_resource_bundle_targets)
-            target.supported_test_types.each do |test_type|
+            target.test_specs.each do |test_spec|
+              spec_consumer = test_spec.consumer(target.platform)
+              test_type = spec_consumer.test_type
               path = target.xcconfig_path(test_type.to_s)
```

Comment:
this will need an update to make the path specific to the test spec?

Reply:
Yes but in a future PR. Not on this one.